packages/builtin/builtin.pony

```diff
@@ -0,0 +1,12 @@
+"""
+# Builtin package
+
+The builtin package is home to the following standard library members:
+
+1. Types the compiler needs to know exist, such as None.
+2. Types with "magic" internal workings that must be supplied directly by the
+compiler, such as U32.
+3. Any types needed by others in builtin.
+
+The public types that are defined in this package will always be in scope for every Pony source file. For details on specific packages, see their individual entity entries.
```

Comment:
Can you hard-wrap this line at 80 chars?

Our convention is to wrap hard-wrap `.pony` files and soft-wrap `.md` files.